test/e2e/specs/wp-calypso-gutenberg-page-editor-spec.js

```diff
@@ -602,26 +602,4 @@ describe( `[${ host }] Calypso Gutenberg Editor: Pages (${ screenSize })`, funct
 			await driverHelper.ensurePopupsClosed( driver );
 		} );
 	} );
-
-	/* Temporarily disabling this test until we smooth out the integration of Gutenberg 7.7.1 See: #40078 */
-
-	describe.skip( 'Use the Calypso Media Modal: @parallel', function () {
```

Comment:
Disabled in #40239

Reply:
Thank you @scinos 

Sorry that I left that in there.